chronicle-front/src/app/components/ticket-add/ticket-add.component.ts

```diff
@@ -0,0 +1,153 @@
+import { Component, Directive, Input, OnInit } from '@angular/core';
+import { AbstractControl, NG_VALIDATORS, ValidationErrors, Validator, ValidatorFn, Validators } from '@angular/forms';
+import { Ticket } from 'src/app/models/Ticket';
+import { AuthService } from 'src/app/services/auth.service';
+import { TicketService } from 'src/app/services/ticket.service';
+
+@Component({
+  selector: 'app-ticket-add',
+  templateUrl: './ticket-add.component.html',
+  styleUrls: ['./ticket-add.component.css']
+})
+
+export class TicketAddComponent implements OnInit {
+ _zoomURL:string ='';
```

Comment:
Is there some reason that you have a Ticket model but declare a lot of the individual pieces of data here? This seems wasteful and unnecessary as you could have just used your ticket object below.

Reply:
We use those for validation.